x11-themes/papirus-icon-theme/papirus-icon-theme-20210601.ebuild

```diff
@@ -20,7 +20,3 @@ src_install() {
 	doins -r ePapirus Papirus{,-Dark,-Light}
 
 }
-
-pkg_preinst() { gnome2_icon_savelist; }
-pkg_postinst() { gnome2_icon_cache_update; }
-pkg_postrm() { xdg_icon_cache_update; }
```

Comment:
Dropped on https://github.com/gentoo/gentoo/pull/19920 because they are not requested anymore